tests/Data/SimulatorTest.elm

```diff
@@ -36,12 +36,13 @@ convertToTests db sectionOrSample =
         Sample.Section title samples ->
             describe title (List.map (convertToTests db) samples)
 
-        Sample.Sample title { query, cch, fwe } ->
+        Sample.Sample title { query, fwe } ->
             describe title
-                [ query
-                    |> expectImpact db (Impact.trg "cch") (Unit.impactToFloat cch)
-                    |> asTest "climate change"
-                , query
+                [ -- FIXME: update samples and uncomment
+                  --     query
+                  --     |> expectImpact db (Impact.trg "cch") (Unit.impactToFloat cch)
+                  --     |> asTest "climate change"
+                  query
```

Comment:
Do you want to fix that before merging?

Reply:
Yes, I'm just waiting for @paulboosz confirmation that computations are correct. Then i'll update all tests with new results and uncomment this.

Reply:
@Got validation from @paulboosz over Mattermost that we can land this. I've uncommented and updated simulation score tests in https://github.com/MTES-MCT/wikicarbone/pull/61/commits/7dd4a3ec350af2abf29967afd78c743cb972fc68.